packages/services/emails/src/environment.ts

```diff
@@ -156,7 +158,7 @@ const emailProviderConfig =
           pass: email.EMAIL_PROVIDER_SMTP_AUTH_PASSWORD,
         },
         tls: {
-          rejectUnauthorized: email.EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED,
+          rejectUnauthorized: !(email.EMAIL_PROVIDER_SMTP_REJECT_UNAUTHORIZED === '0'),
```

Comment:
it's `true` by default, right?

Reply:
Yes, by default it's true
https://nodejs.org/api/tls.html#tlscreateserveroptions-secureconnectionlistener
`rejectUnauthorized - If not false the server will reject any connection which is not authorized with the list of supplied CAs. This option only has an effect if requestCert is true. Default: true.`